### PR TITLE
[qt5-base] use supports expression

### PR DESCRIPTION
**Pull request by autoantwort**
From portfile.cmake
```cmake
if(WITH_MYSQL_PLUGIN AND NOT VCPKG_TARGET_IS_WINDOWS)
    message(WARNING "${PORT} is currently not setup to support feature 'mysqlplugin' on platforms other than windows. Feel free to open up a PR to fix it!")
endif()
```

### CODE REVIEW

**Comment by dg0yt** [comment]
Is this right? There was only a warning. And the issue is probably unrelated to upstream support.

**Comment by Neumann-A** [comment]
The warning is probably due to missing (system) deps as ports instead of a hard blocker.

File: ports/qt5-base/vcpkg.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     },
     "mysqlplugin": {
       "description": "Build the sql plugin for connecting to mysql databases",
+      "supports": "windows",
```
Comment (Neumann-A) [inline ports/qt5-base/vcpkg.json:52]: see https://github.com/qt/qtbase/blob/ab28ff2207e8f33754c79793089dbf943d67736d/src/plugins/sqldrivers/configure.json#L69-L78

**Comment by autoantwort** [comment]
I have created #28426

**Pull request closed without merging**